R/metacore.R

```diff
@@ -372,7 +378,7 @@ metacore <- function(ds_spec = tibble(dataset = character(), structure = charact
          })
       names(replaced) <- to_replace %>% map_chr(~unique(.$dataset))
       list2env(replaced, environment())
-      }
+   }
    MetaCore$new(ds_spec, ds_vars, var_spec, value_spec, derivations, codelist, supp, quiet)
```

Comment:
I believe this also needs to be wrapped in `quiet_if_true`. If you look at the original example given in the issue, when `metacore()` is called directly the warnings still persist even if `quiet = TRUE`.

```suggestion
quiet_if_true(MetaCore$new(ds_spec, ds_vars, var_spec, value_spec, derivations, codelist, supp, quiet))
```